### PR TITLE
3.4 Closure actions deprecation

### PR DESCRIPTION
**Pull request by betocantu93**
Deprecations Added in 3.4
Use closure actions instead of `sendAction`

https://emberjs.com/deprecations/v3.x#toc_ember-component-send-action

### CODE REVIEW

**Comment by avdept** [comment]
Any progress on merging this?

**Comment by miguelcobain** [comment]
@betocantu93 Thank you very much for this.
Would it be too much to ask if we could use [ember-invoke-action](https://github.com/martndemus/ember-invoke-action) addon?

The addon has a functional utility, just like you made here, and also has a mixin. I'd prefer the functional utility because it is better aligned with ES6 classes which don't support mixins. I'm planning to migrate ember-paper soon.

Using this addons promotes using shared solutions and it is already tested against many ember versions.
Let me know if you're ok to do it.

**Comment by betocantu93** [comment]
@miguelcobain Interesting, nice addon. Excellent news about ember-paper migration, let us know if you need help with that - also I was planning an attempt to get ember-paper run without any jquery -  and sure, I can do the addon thing, I'll push to this branch I guess.

File: addon/components/paper-chips.js [hunk 1]
```diff
@@ -170,7 +171,7 @@ export default Component.extend({
 
     noUnselected(old, event) {
       if (['Backspace', 'Delete', 'Del', 'ArrowLeft', 'Left', 'ArrowRight', 'Right'].includes(event.key)) {
-        this.sendAction('keyDown', event);
+        invokeAction(this, 'keyDown', item);
```
Comment (marcemira) [inline addon/components/paper-chips.js]: this should be `event` instead of `item`

Comment (miguelcobain) [reply addon/components/paper-chips.js]: Exactly. This is the error on travis.
Linting tests are useful after all, huh.

**Comment by miguelcobain** [comment]
Thanks a lot for this!

**Pull request merged**